lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -701,6 +694,21 @@ sub jsonrpc_validate {
                 }
             };
         }
+
+        my %locale = $self->{config}->Language_Locale_hash();
+
+        # NOTE: Default to first language in config
+        my $language = $jsonrpc_request->{params}->{language} // $self->{config}->{_LANGUAGE_locale}[0];
+
+        # NOTE: temporary workaround to not break everything
+        if (exists $jsonrpc_request->{params}->{language} && !exists $method_schema->{properties}->{language}) {
+            delete $jsonrpc_request->{params}->{language};
+        }
+
+        # TODO: Language validation
+        $ENV{LANGUAGE} = $locale{$language};
+        setlocale( LC_MESSAGES, $locale{$language} );
```

Comment:
I think `$language` here should be a full locale string, e.g. `en_US.UTF-8`.

Reply:
The language code to full locale conversion is performed by the `%locale` hash (see comment above).